grammar.js

```diff
@@ -292,6 +292,7 @@ module.exports = grammar({
     _type_identifier: $ => alias($.identifier, $.type_identifier),
     tag_identifier: $ => seq("#", $.identifier),
     privileged_identifier: $ => seq("@", token.immediate($.identifier)),
+    proj_identifier: $ => /[0-9]+/,
```

Comment:
Please preserve the `token.immediate` here:
```suggestion
    proj_identifier: $ => token.immediate(/[0-9]+/),
```

as `(1, 2). 0` is not valid Motoko.